src/app/components/ReadTime/index.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReadTime from '.';
+import readme from './README.md';
+
+const Component = ({ readTime }: { readTime: number }) => (
+  <ReadTime readTime={readTime} />
+);
+
+export default {
+  title: 'Components/ReadTime',
+  Component,
+  parameters: {
+    docs: { readme },
+  },
+};
+
+export const ExampleReadTimeOver1Minute = () => <ReadTime readTime={7} />;
+export const Example1MinuteReadTime = () => <ReadTime readTime={1} />;
```

Comment:
For consistency, there should be one named export called `Example`, which is the most common use case for that component.

All other stories can have a descriptive name (and should not include the word "Example")

```suggestion
export const Example = () => <ReadTime readTime={7} />;
export const 1MinuteReadTime = () => <ReadTime readTime={1} />;
```